.github/workflows/node-ci.yml

```diff
@@ -82,7 +82,7 @@ jobs:
             arch: arm64
           - runs-on: macos-14-large
             arch: x86_64
-          - runs-on: windows-2022
+          - runs-on: windows-latest
```

Comment:
OK with me but might start to fail if a new Windows version is out.

Reply:
Hmmm maybe. It's because it was `-latest` in `windows-ci.yml`, and in other places I see `ubuntu-latest` too, so I was thinking it was better to make every Windows as latest. But I can do the opposite, make every Windows 2022, what do you think?

Reply:
Agree, you are right. I will have to make some corrections, so `windows-2022` will also be the standard and will be in the next commit.